boards/arm/96b_carbon/Kconfig.defconfig

```diff
@@ -10,4 +10,47 @@ if BOARD_96B_CARBON
 config BOARD
 	default 96b_carbon
 
+if I2C
+
+config I2C_1
+	def_bool y
+
+config I2C_2
+	def_bool n
+
+config I2C_3
+	def_bool n
+
```

Comment:
do we really need I2C_2 & I2C_3 here, aren't they default 'n' in drivers/i2c/Kconfig.  Also there isn't any I2C_3 defined anywhere.